torrent/session_torrent.go

```diff
@@ -42,6 +42,18 @@ func (t *Torrent) Name() string {
 	return t.torrent.Name()
 }
 
+// RootDirectory of the torrent.
+// The directory that contains the files in the torrent.
+func (t *Torrent) RootDirectory() string {
+	return t.torrent.RootDirectory()
+}
+
+// The files in the torrent.
+// The names of the files are relative to the root directory.
+func (t *Torrent) Files() ([]string, error) {
```

Comment:
Can you change this method's name to `FileNames()`? We may introduce `Files()` in future for returning the actual `*os.File`s.

Reply:
maybe file paths? since file name could be confused with just the name of the file, without any info about its location